### DISCUSSION TITLE
Outbound link tracking not working with javascript

### DISCUSSION THREAD

**Discussion by slubbodan** [plausible/analytics#1054] [Anything Else]
Hi, I'm trying out Plausible and the outbound link tracking. It's working perfectly with "standard" links, but in our CMS we also have a module where links are "embedded" in javascript. These links doesn't seem to be tracked, and i wonder if there is any possibility to make this happen?
Here is an example.
https://nai.uu.se/news-and-events/news/2021-05-06-pandemic-makes-life-hard-for-rural-distributors.html
The link to World Health Organisation in the facts box on Zambia is tracked.
The links at the bottom of the page (in the section Resources from the library) is not tracked.

**Comment by ukutaht** [comment]
Hey, thanks for giving Plausible a try!

Outbound link tracking is intended to work only with standard HTML links. If you're using Javascript to navigate visitors, you can also use Javascript to trigger a [custom event](https://plausible.io/docs/custom-event-goals) in Plausible before the navigation. Sorry I don't have a better solution at the moment.

**Reply by slubbodan** [reply]
Thanks for the reply! Do you have any tips on how to trigger the custom event? I'm not really the technical guy.... the javascript always looks like this:

<a class="nai-related-item" 

then there is 

href="javascript:void(window.open('the url here', etc

**Reply by ukutaht** [reply]
No worries. Try changing the href attribute from `javascript:void(window.open('url here'))` to something like `javascript:void(plausible('Outbound link: Click', {url: 'url here'}); window.open('url here'))`.

First, you'll also need to include a line next to your Plausible script tag to make custom events available: https://plausible.io/docs/custom-event-goals#1-trigger-custom-events-with-javascript-on-your-site
